src/loadMessages.js

```diff
@@ -0,0 +1,11 @@
+export default function loadMessages(lang) {
+  const wrapped = import(`./messages/${lang}`)
+
+  if (typeof window !== 'undefined') {
+    return wrapped.then(msgs => msgs.default)
+  }
+
+  /* eslint-disable global-require, import/no-dynamic-require, prefer-template */
+  // return Promise.resolve(require('./messages/' + lang).default)
+  return __webpack_require__(wrapped.resolve()).default
+}
```

Comment:
This code works pretty fine. Only remaining issue is to fill the rendered chunks on server side correctly so that "en.js" is added to the HTML script section.